prepare-cluster-dev.sh

```diff
@@ -0,0 +1,40 @@
+
+#!/bin/bash
+
+# Exit if any of the intermediate steps fail
+set -e
+# set bash debug
+set -x
+
+# install OpenShift GitOps
+oc apply -k https://github.com/cloud-native-toolkit/deployer-cluster-prep//argocd-operator/?ref=develop --insecure-skip-tls-verify=true
+while ! oc wait crd applications.argoproj.io --timeout=1800s --for=condition=Established  --insecure-skip-tls-verify=true; do sleep 30; done
+while ! oc wait deployment/cluster -n openshift-gitops --timeout=1800s --for=condition=Available  --insecure-skip-tls-verify=true; do sleep 30; done
+while ! oc wait deployment/kam -n openshift-gitops --timeout=1800s --for=condition=Available  --insecure-skip-tls-verify=true; do sleep 30; done
+
+oc apply -k https://github.com/cloud-native-toolkit/deployer-cluster-prep//argocd-instance/?ref=develop --insecure-skip-tls-verify=true
+
+# install OpenShift Pipelines
+oc apply -k https://github.com/cloud-native-toolkit/deployer-cluster-prep//openshift-pipelines-operator/?ref=develop --insecure-skip-tls-verify=true
+while ! oc wait crd pipelines.tekton.dev --timeout=1800s --for=condition=Established  --insecure-skip-tls-verify=true; do sleep 30; done
+while ! oc wait crd pipelineruns.tekton.dev --timeout=1800s --for=condition=Established  --insecure-skip-tls-verify=true; do sleep 30; done
+while ! oc wait crd tasks.tekton.dev --timeout=1800s --for=condition=Established  --insecure-skip-tls-verify=true; do sleep 30; done
+while ! oc wait --for=condition=ready TektonPipelines/pipeline --timeout=1800s --insecure-skip-tls-verify=true; do sleep 30; done
+while ! oc wait --for=condition=Ready TektonAddon/addon  --timeout=1800s --insecure-skip-tls-verify=true; do sleep 10; done
```

Comment:
Do you need to include a Kustomize patch to prevent 1.11 from being installed to use Cluster Tasks, which are provided in the tekton add-on @davidstacy ?